metrics_test.go

```diff
@@ -104,4 +104,12 @@ func getVolumeMetrics(t *testing.T) {
 		}
 		fmt.Println("================================")
 	}
+
+	// Checking GetCapacity function
+	systemCapacityResult, testErr := testConf.metricsAPI.GetCapacity(ctx)
+	if testErr != nil {
+		t.Fatal(testErr)
+		return
+	}
+	fmt.Println(systemCapacityResult)
```

Comment:
Can we remove the print statement from here?

Reply:
@karthikk92 Since we are not asserting anything in the integration tests, printing the response to make sure we are getting the same response as expected.

Reply:
Sure